benchmarks/kernels/benchmark_sgmv_triton.py

```diff
@@ -0,0 +1,127 @@
+import torch
+import triton
+
+from tests.lora.test_sgmv_triton import MAX_TEST_POWER, setup_
```

Comment:
Ideally, benchmark code shouldn't depend on the `tests` module -- I had to manually set my PYTHONPATH to run this. Duplicating the `setup_` function might be a better way to do it in this case, as much as I hate to suggest it.

Reply:
I could put the function somewhere in main vllm rather than in the test module